contribs/gmf/src/controllers/bootstrap.js

```diff
@@ -1,12 +1,13 @@
+// FIXME DocumentTouch is deprecated, see:
+// https://developer.mozilla.org/en-US/docs/Web/API/DocumentTouch
 /* global DocumentTouch */
 
 import $ from 'jquery';
 import angular from 'angular';
 
-
 function bootstrap(module) {
-  const interface_ = $('meta[name=interface]')[0].attributes.content.value;
-  const dynamicUrl_ = $('meta[name=dynamicUrl]')[0].attributes.content.value;
+  const interface_ = $('meta[name=interface]')[0].getAttribute('content');
```

Comment:
Not sure it works: if I run: `$('meta[name=interface]')[0].getAttribute('content')` on https://camptocamp.github.io/ngeo/master/examples/contribs/gmf/apps/desktop.html/

I get the following error:
```
Uncaught TypeError: Cannot read property 'getAttribute' of undefined
    at <anonymous>:1:29
```

Reply:
Should be:
```suggestion
  const interface_ = $('meta[name=interface]').getAttribute('content');
```

Reply:
I noticed that too.  From what I understand, if you execute the code in a console then it doesn't work.  Try putting a breakpoint after and it works.

I suspect that the scope of `$` is not the same while we execute, because the code does with `[0]` and it was like this before and it worked.  I'm not sure if we should remove it (I'll try).

Reply:
As expected, if I remove the `[0]`I get a thrown JavaScript error.  I won't touch this, because it works already.